src/providers/ldap/sdap_autofs.c

```diff
@@ -250,50 +260,211 @@ sdap_autofs_handler_send(TALLOC_CTX *mem_ctx,
         goto immediately;
     }
 
-    tevent_req_set_callback(subreq, sdap_autofs_handler_done, req);
+    tevent_req_set_callback(subreq, sdap_autofs_enumerate_handler_done, req);
 
     return req;
 
 immediately:
-    dp_reply_std_set(&state->reply, DP_ERR_DECIDE, ret, NULL);
+    if (ret == EOK) {
```

Comment:
I think we cant reach this line with ret == EOK

Reply:
Fixed.